commons/helpers/pods_helper.py

```diff
@@ -526,3 +513,30 @@ def get_all_cluster_processes(self, pod_name, container_name):
                                  decode=True)
         process_list = resp.splitlines()
         return process_list
+
+    def get_deployment_name(self, pod_prefix=None):
+        """
+        Get deployment name using kubectl get deployment
+        :param pod_prefix: Pod prefix(optional)
+        return: list
+        """
+        resp_node = self.execute_cmd(cmd=commands.KUBECTL_GET_DEPLOYMENT, read_lines=True,
+                                    exc=False)
+
+        resp = [each.split()[0] for each in resp_node]
+        deploy_list = resp
+        if pod_prefix is not None:
+            deploy_list = [each for each in resp if pod_prefix in each]
+        return deploy_list
+
+    def apply_k8s_deployment(self, file_path: str):
```

Comment:
Can you check if you can use recover_deployment_k8s()?

Reply:
apply_k8s_deployment is written for applying modified yaml. 
In recover_deployment_k8s we are creating new pods. 
Not sure if kubectl create will work on already existing pod.